### PR TITLE
Category branch

### LINKED ISSUE #22: Set up category page

### PR DESCRIPTION
**Pull request by motuncoded**
Set up category page 

resolves #22

### CODE REVIEW

File: src/app/categories/page.tsx [hunk 1]
```diff
@@ -1,7 +1,42 @@
+import Link from "next/link";
+import categories from "../json/categories.json";
+import Image from "next/image";
+
 function CategoriesPage() {
   return (
     <div>
-      <p>Categories page</p>
+      <div className=" flex justify-center items-center  h-32 w-[calc(100% - 2rem)] rounded">
+        <h2 className="text-4xl font-bold bg-gradient-to-r from-blue-500 via-yellow-400 to-orange-400 bg-clip-text text-transparent">
```
Comment (Asylcreek) [inline src/app/categories/page.tsx:9]: Why not `h1`?

File: src/app/components/Footer.tsx [hunk 2]
```diff
@@ -17,15 +17,14 @@ interface ApiResponse {
 
 const FooterLogo = () => (
   <div className="flex flex-col pt-4">
-    <h3 className="flex items-center">
+    <h3 className="flex items-center mb-4">
```
Comment (Asylcreek) [inline src/app/components/Footer.tsx:20]: This can be a `p`

File: src/app/components/Footer.tsx [hunk 3]
```diff
@@ -164,30 +163,39 @@ const Subscribe = () => {
   }, [subscribeMessage]);
   return (
     <section className="mt-4" aria-label="Subscribe ">
-      <h3>Subscribe for our newsletter.</h3>
-      <p className="text-[12px] w-[250px] text-gray-800">
+      <h3 className="font-bold">Subscribe for our newsletter.</h3>
```
Comment (Asylcreek) [inline src/app/components/Footer.tsx:166]: This can be a `p`

File: src/app/components/Footer.tsx
Comment (Asylcreek) [inline src/app/components/Footer.tsx:1]: The use of heading tags is supposed to be hierarchical. You should not use `h2` if there is no `h1` on your page. And you should never skip a tag, meaning that if there is an `h4` on your page, there must already be an `h1`, `h2`, and `h3`

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: src/app/categories/page.tsx (hunk 1 above)
Comment (motuncoded) [reply src/app/categories/page.tsx:9]: ok

File: src/app/components/Footer.tsx
Comment (motuncoded) [reply src/app/components/Footer.tsx:1]: Noted...

**Review by Asylcreek — APPROVED** [review]

**Pull request merged**